### DISCUSSION TITLE
SensESP and freeRTOS

### DISCUSSION THREAD

**Discussion by pwm802** [hatlabs/discussions#51] [Q&A]
Hello,
I am wondering if anyone is using freeRTOS with SensESP?  Any thoughts or examples would be greatly appreciated.

**Comment by mairas** [comment]
Given that the ESP32 Arduino Core runs on FreeRTOS, it already is. FreeRTOS tasks are used internally for some purposes, and in the examples folder there is an example for using tasks in an end user application.

**Comment by pwm802** [comment]
Thanks, I am new to freeRTOS and wondered why I didn't see more tasks being started in the example apps.

Sent via Outlook for Android
________________________________
